src/drivers/distance_sensor/tfmini/tfmini.cpp

```diff
@@ -173,7 +174,7 @@ class TFMINI : public device::CDev
 extern "C" __EXPORT int tfmini_main(int argc, char *argv[]);
 
 TFMINI::TFMINI(const char *port, uint8_t rotation) :
-	CDev("TFMINI", RANGE_FINDER0_DEVICE_PATH),
+	CDev(NAME, DEVICE_PATH),
```

Comment:
@dagar 
>What's wrong with defaulting to `RANGE_FINDER0_DEVICE_PATH` and letting the command line override?

I guess you're talking about this?

Can we have several range finders this way?